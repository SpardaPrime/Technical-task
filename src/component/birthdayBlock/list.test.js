import List from './list';
import renderer from 'react-test-renderer';

describe('component list ',()=>{

    test('test 1',()=>{
            const newusers = [];
            let componnet = renderer.create(<List users={newusers}/>).toJSON();
            expect(componnet).toBeNull();
    });

});
