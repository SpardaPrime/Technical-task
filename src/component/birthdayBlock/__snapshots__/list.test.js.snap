// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component list  test 1 1`] = `null`;

exports[`component list  test 2 1`] = `null`;
